src/plugins/data/public/query/filter_manager/lib/mappers/map_phrase.ts

```diff
@@ -17,6 +18,10 @@ import {
 } from '@kbn/es-query';
 import { FieldFormat } from '@kbn/field-formats-plugin/common';
 
+const isEmptyString = (value: string | number | boolean | undefined) => {
+  return typeof value === 'string' && value === '';
```

Comment:
NIT: `value === ''` already checks the type, the first `typeof value === 'string'` condition is redundant.